NitroxClient/GameLogic/Spawning/DefaultEntitySpawner.cs

```diff
@@ -28,6 +28,12 @@ public Optional<GameObject> Spawn(Entity entity, Optional<GameObject> parent)
             gameObject.transform.localRotation = entity.Rotation;
             GuidHelper.SetNewGuid(gameObject, entity.Guid);
             gameObject.SetActive(true);
+            // Makes movable objects movable... we can probably do this before the server sends the spawner packet?
+            if(gameObject.GetComponent<Rigidbody>() ==  null)
+            {
+                gameObject.AddComponent<Rigidbody>();
```

Comment:
I'm not sure the implications of having all entities spawning with a rigid body.  I think we should dive deeper into why the prefabs are not being initialized with the correct components.  Maybe do a diff from the result of `CraftData` and `PrefabDatabase` above to see if there are any differences for affected tech types.

Reply:
Yep I wasn't sure either I tested a fair bit after and it seems everything is fine. This can be used as a quick fix for story line progression safely if people want that. 

I beileve we are diving into proper initialization now along with sever only spawning. Can we create a master branch for this as it's quite a major overhaul and I think it would be nice to have some clarity on master that this is being worked on. 

Just my thoughts feel free to close PR anyway and I will carry on with looking for better initialization methods.

Reply:
I'm fine merging this in as a temporary fix.  Let's create an issue so we remember to remove it.  

Also, let's wait to see what the actual fix for spawning is.  I think there are two issues: 1) incorrect prefab used for loading or incorrect bootstrapping 2) A missing source of parsed files.  If that is the case, these are probably minor changes.  We'll see when we get there though.